test/main.spec.js

```diff
@@ -0,0 +1,14 @@
+import test from 'tape'
+import addTwoNumbers from './../src/main'
```

Comment:
(see node-sdk example)

The import then will be `import addTwoNumbers from 'main'`, the path to `src` is then implicit in the `NODE_PATH`